src/js/components/Box/StyledBox.js

```diff
@@ -200,6 +201,20 @@ const roundStyle = css`
   border-radius: ${props => ROUND_MAP[props.round] || props.theme.global.edgeSize[props.round]};
 `;
 
+const responsiveStyle = css`
+  ${props => palm(`
+    flex-direction: column;
+
+    ${props.justify === 'center' && 'align-items: stretch;'}
+    ${props.reverse && 'flex-direction: column-reverse'}
+
+    > * {
```

Comment:
This doesn't seem like a styled-components way of doing things. I've been wanting to avoid wildcard selectors like this.